mica-webapp/src/main/resources/_templates/studies.ftl

```diff
@@ -154,9 +154,51 @@
 
 <#include "libs/scripts.ftl">
 <script>
-    $(function () {
-        $("#${title}").DataTable(dataTablesDefaultOpts);
-    });
+  const Mica = {
```

Comment:
JS script is only acceptable at this place when it is simple. You should move this code to a separate template so that user does not break logic when changing page's template theme.